src/Unleash/UnleashSettings.cs

```diff
@@ -63,11 +63,11 @@ public class UnleashSettings
         public TimeSpan FetchTogglesInterval { get; set; } = TimeSpan.FromSeconds(30);
 
         /// <summary>
-        /// Gets or sets the interval in which metriccs are sent to the server. When null, no metrics is sent.
+        /// Gets or sets the interval in which metrics are sent to the server. When null, no metrics is sent.
```

Comment:
Thanks for fixing the spelling here! Got a little grammar fix while we're here

```suggestion
        /// Gets or sets the interval in which metrics are sent to the server. When null, no metrics are sent.
```